### ISSUE TITLE
Manual alter weights of GLM

### ISSUE THREAD

**Issue by skjerns** [X-DataInitiative/tick#471]
I want to extract coefficients of a GLM, then alter them, and set them again as the weight matrix.

However, `weights` is readonly. 

```
data_x = np.random.rand(512, 300)
data_y = np.random.randint(0, 2, 512)
clf = LogisticRegression()
clf.fit(data_x, data_y)

clf.weights *= 3.14

# AttributeError: weights is readonly in LogisticRegression
```

Is there any way to manually set weights to a GLM?

**Comment by PhilipDeegan** [comment]
you can set them like this if you want to https://github.com/X-DataInitiative/tick/blob/master/tick/base/learner/learner_glm.py#L206

**Comment by Mbompr** [comment]
Indeed, these weights are supposed to be read-only. If you want to set this attribute manually, you can run `clf._set("weights", clf.weights * 3.14)`. But do it at your own risks :)
